src/dsp_tools/utils/excel_to_json_resources.py

```diff
@@ -56,12 +70,35 @@ def _row2resource(row: pd.Series, excelfile: str) -> dict[str, Any]:
         location_of_sheet=f"Sheet '{name}' in file '{excelfile}'"
     )
 
+    # validation
+    # 4 cases:
+    #  - column gui_order absent
+    #  - column gui_order empty
+    #  - column gui_order present but not properly filled in (missing values / not integers)
+    #  - column gui_order present and properly filled in
+    all_gui_order_cells = [x for x in details_df.get("gui_order", []) if x]
```

Comment:
I'm wondering if `gui_order` should be an attribute that can be set by the user. I think it would be more user-friendly to just take the order given in the Excel sheet (which is already the case). I can't imagine a use case where a user actually _wants_ to set the gui order him/herself other than the order in the Excel sheet.

Reply:
This feature was requested by Noémi, she would find it useful.